README.md

```diff
@@ -83,6 +83,6 @@ Learn more about Catalyst at [catalyst.dev](https://catalyst.dev).
 
 ![-----------------------------------------------------](https://storage.googleapis.com/bigcommerce-developers/images/catalyst_readme_hr.png)
 
-> [!IMPORTANT]
-> If you just want to build a storefront, start with the [CLI](#quickstart) which will install the Next.js application in [/core](/core/).
+> **Important:**
+> If you just want to build a storefront, start with the [CLI](#quickstart) which will install the Next.js application in [/core](https://github.com/bigcommerce/catalyst/tree/main/core).
```

Comment:
@matthewvolk shouldn't this be the root monorepo now instead of `/core`?

Reply:
Correct! Let's maybe change it to:

```suggestion
> If you just want to build a storefront, start with the [CLI](#quickstart) which will install the [entire Catalyst repository](https://github.com/bigcommerce/catalyst) locally.
```